package.json

```diff
@@ -19,20 +19,20 @@
     "posttest": "npm run lint"
   },
   "dependencies": {
-    "async": "^2.1.5",
-    "debug": "^4.0.1",
-    "loopback-connector": "^4.0.0",
-    "oracledb": "^3.0.0",
-    "strong-globalize": "^4.1.2"
+    "async": "^2.6.3",
```

Comment:
I'm not sure if updating this to use a more recent minor version (2.1.5 -> 2.6.3) is necessary. Similar to other changes too. But others are good with it, I'm fine with it too.

Reply:
I mostly wanted to update `strong-globalize` and `eslint*` and just updated these along with them, but I agree it's not necessary. Do you want me to revert them or just keep them?

Reply:
I’m thinking… since there's nothing wrong to make the change and it’s already done, we might as well just land the way it is now.  Not sure if others object.